src/main/java/uk/gov/hmcts/ccd/endpoint/std/CallbackEndpoint.java

```diff
@@ -32,7 +31,6 @@ public CallbackEndpoint(final PrintableDocumentListOperation printableDocumentLi
         this.printableDocumentListOperation = printableDocumentListOperation;
     }
 
-    @Transactional
     @RequestMapping(value = "/jurisdictions/{jid}/case-types/{ctid}/documents", method = RequestMethod.POST)
```

Comment:
do we need POST for this request

Reply:
mmmm yes we need evaluate it . not 100 % sure .